code/modules/mob/living/simple_animal/friendly/iriska.dm

```diff
@@ -0,0 +1,203 @@
+				if(!(mob in despised))
+					tolerate(mob)
+
+			else if(ishuman(snack.loc) && Adjacent(src, get_turf(snack)) && prob(15))
+				beg(snack, snack.loc)
+	else
+		scan_interval = max(min_scan_interval, min(scan_interval+1, max_scan_interval))
+
+/mob/living/simple_animal/iriska/proc/react_to_mob()
+	for(var/mob/living/M in oview(src, 1))
+		if (M.stat != DEAD)
+
+			if(iscorgi(M))
+				if(prob(5)) visible_emote("pointedly ignores [M].")
+
+			else if(iscat(M))
+				var/verb = pick("meows", "mews", "mrowls")
+				if(prob(5)) visible_emote("[verb] at [M].")
+
+			else if(ishuman(M))
+				if(M in tolerated)
+					if(prob(2)) say("Meoow!")
+
+				else if ((M.job == befriend_job) && !(M in despised)) // Recognize captain
+					tolerated |= M
+					visible_emote("looks at [M] with a hint of respect.")
+
+				else
+					assert_dominance(M)
+
+			else
+				assert_dominance(M)
+	return
+
+/mob/living/simple_animal/iriska/proc/assert_dominance(var/mob/target_mob)
+	if(prob(15)) say("HSSSSS")
+	if(!Adjacent(target_mob))
+		return
+	if(isliving(target_mob))
+		var/mob/living/L = target_mob
+		L.attack_generic(src,rand(melee_damage_lower,melee_damage_upper),attacktext)
+		return L
+	if(istype(target_mob,/mob/living/exosuit))
+		var/mob/living/exosuit/M = target_mob
+		M.attack_generic(src,rand(melee_damage_lower,melee_damage_upper),attacktext)
+		return M
+	if(istype(target_mob,/obj/machinery/bot))
+		var/obj/machinery/bot/B = target_mob
+		B.attack_generic(src,rand(melee_damage_lower,melee_damage_upper),attacktext)
+		return B
+
+/mob/living/simple_animal/iriska/proc/despise(mob/living/carbon/human/M as mob)
+	despised |= M
+	if(M in tolerated)
+		tolerated -= M
+
+/mob/living/simple_animal/iriska/proc/tolerate(mob/living/carbon/human/M as mob)
+	if(!(M in tolerated) && prob(30))
+		visible_emote("looks at [M] approvingly.")
+		tolerated += M
+
+/mob/living/simple_animal/iriska/attackby(var/obj/item/O, var/mob/user)
+	. = ..()
+	if(O.force)
+		despise(user)
+
+/mob/living/simple_animal/iriska/attack_hand(mob/living/carbon/human/M as mob)
+	. = ..()
+	if(M.a_intent == I_HURT)
+		despise(M)
+	if((M.a_intent == I_HELP) && (M in tolerated))
+		if(prob(15)) say("PRRRR")
+
+/mob/living/simple_animal/iriska/bullet_act(var/obj/item/projectile/proj)
+	. = ..()
+	despise(proj.firer)
+
+/mob/living/simple_animal/iriska/hitby(atom/movable/AM)
+	. = ..()
+	despise(AM.thrower)
+
+/mob/living/simple_animal/iriska/death(gibbed, deathmessage = "dies!")
+	destroy_lifes()
+	.=..()
+
+	snack = null
+	return ..(gibbed,deathmessage)
+
+/mob/living/simple_animal/iriska/proc/destroy_lifes()
+	for(var/mob/living/carbon/human/H in despised)
+		H.sanity.insight = 0
+		H.sanity.environment_cap_coeff = 0
+		H.sanity.negative_prob += 30
+		H.sanity.positive_prob = 0
+		H.sanity.level = 0
+		for(var/stat in ALL_STATS)
+			H.stats.changeStat(stat, -10)
+		to_chat(H, SPAN_DANGER("THE SHADOWS SEEM TO LENGTHEN AND WALLS ARE GETTING CLOSER. SHIP ITSELF WANTS YOU DEAD."))
```

Comment:
"The shadows seem to lengthen, the walls are closing in. The ship itself wants you dead."

Reply:
Can it be uppercase? That line easily missed, just doesn't stand out from other red logs, and those expected if player killing the cat.

Reply:
I think there's a span_ that's more intense than span_danger. Might be what you're looking for.

Reply:
Seems like span_danger is most intense, or I'm just looking wrong.